src/services/config.js

```diff
@@ -49,6 +52,20 @@ const config = {
     return process.env.CHECKLY_ACCOUNT_ID || this.data.get('accountId')
   },
 
+  getProjectId() {
+    try {
+      const parsedSettings = YAML.parse(
+        fs.readFileSync(
+          path.join(process.cwd(), '.checkly/settings.yml'),
+          'utf8'
+        )
+      )
+      return parsedSettings.projectId
+    } catch (e) {
+      return undefined
```

Comment:
I think `null` is more accurate

Reply:
Does it matter in this case? The only place its being used atm is in the new `projects.delete()` services/api method being added in this PR as well, i.e. at `src/modules/projects/delete.js:10`

Reply:
It's more a philosophical concept where I believe that `null` is more intended to represent the intentional absence. (IMO)

Reply:
Intentional absence is my new favorite phrase :D

Reply:
Yeah I was thinking about this a bit actually, I wanted the projectId to be there, but not _be_ anything, so I went with undefined. But it really doesn't matter to me, I'll change it to `null` :+1: